src/cli.ts

```diff
@@ -29,15 +33,19 @@ Please specify DATABASE_URL to run this CLi. Try the following:
 
 const db = new Kysely({
   dialect: new PostgresDialect({
-    connectionString: DATABASE_URL,
+    pool: new Pool({
```

Comment:
the newer version of the pg dialect uses a pool instance to specify the connection string

Reply:
Thanks!